setup.py

```diff
@@ -78,7 +78,7 @@
     f"jaxlib=={jax_version}",
     "tomlkit; python_version < '3.11'",
     "scipy<1.13",
-    "numpy<2",
+    "numpy>=2",
```

Comment:
Assuming there are no changes required on our side, and we are backwards compatible, should we allow <2 as well?

Reply:
```suggestion
    "numpy",
```

Reply:
What I wanted was to force numpy>=2 and run ci. I'll keep `numpy>=2` to force it because some requirements.txt have `numpy<2`.

```
    "numpy>=2",
```

Reply:
No worries, I wasn't saying it to change it now, just a general point of discussion for later.